src/main/java/frc/robot/constants/AcquisitionConstants.java

```diff
@@ -1,10 +1,16 @@
 package frc.robot.constants;
 
 public class AcquisitionConstants {
-    public static final int motorPort = -1;
+    public static final int intakeMotorPort = 0;
+    public static final int sushiMotorPort = 0;
+    public static final int acqLaserPort = 0;
 
     public static final double motorSpeed = 0.0;
     public static final double reverseMotorSpeed = 0.0;
     public static final double maxMotorSpeed = 1;
     public static final double minMotorSpeed = -1;
```

Comment:
Delete these constants if we are setting the motors to the constant speeds found below.